src/common/HtmlEditor.svelte

```diff
@@ -60,7 +61,7 @@
 
 <div class="sidebyside">
   <textarea {placeholder} use:textAreaResize bind:value {maxlength} />
-  {#if $domPurify.domPurify != null && value.trim().length > 0}
+  {#if $domPurify.domPurify !== null && typeof $domPurify.domPurify.sanitize === "function" && value.trim().length > 0}
```

Comment:
Because `domPurify` is a store that is sometimes null, I was hitting an error where `sanitize` wasn't a function and it crashed the page. It should be fixed now, but this is an extra check just to be safe.